### PR TITLE
Improve translations

### PR DESCRIPTION
**Pull request by nickvergessen**
After doing the german translation, I found some mistakes.

cc @Ivansss @schiessle

### CODE REVIEW

**Comment by Ivansss** [comment]
LGTM 👍

File: appinfo/app.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 		'order' => 3,
 		'href' => $g->linkToRoute('spreed.page.index'),
 		'icon' => $g->imagePath('spreed', 'app.svg'),
-		'name' => $l->t('Calls'),
+		'name' => $l->t('Video calls'),
```
Comment (jancborchardt) [inline appinfo/app.php]: This name will get way too long when translated. Let’s use just »Calls« here please.

File: templates/settings-admin.php [hunk 2]
```diff
@@ -33,13 +33,13 @@
             <select id="turn_server_protocols" name="turn_server_protocols">
                 <option value="udp,tcp"
                     <?php p($_['turnServerProtocols'] === 'udp,tcp' ? 'selected' : '') ?>>
-                    udp and tcp</option>
+                    <?php p($l->t('udp and tcp')) ?><</option>
```
Comment (jancborchardt) [inline templates/settings-admin.php]: Btw should this be capitalized properly like »UDP and TCP«, »UDP only« etc?

File: templates/settings-personal.php [hunk 3]
```diff
@@ -18,27 +18,27 @@
...
+                   name="turn_password" placeholder="<?php p($l->t('Password')) ?>"
                    value="<?php p($_['turnSettings']['password']) ?>" />
         </p>
         <p>
             <label for="turn_protocols"><?php p($l->t('Protocols')) ?></label>
             <select id="turn_protocols" name="turn_protocols">
                 <option value="udp,tcp"
                     <?php p($_['turnSettings']['protocols'] === 'udp,tcp' ? 'selected' : '') ?>>
-                    udp and tcp</option>
+                    <?php p($l->t('udp and tcp')) ?><</option>
```
Comment (jancborchardt) [inline templates/settings-personal.php]: Same here?

**Comment by jancborchardt** [comment]
@nickvergessen I’d prefer if we continue to name it »Calls« in the apps menu, especially because of length of translations.

Also, what do you think about my other »UDP« and »TCP« capitalization remarks?

File: templates/settings-admin.php (hunk 2 above)
Comment (nickvergessen) [inline templates/settings-admin.php]: I guess so, feel free to fix

**Comment by nickvergessen** [comment]
Honestly I dont want this app to be translated "Anrufe" in german.
That is not what it is.

**Comment by jancborchardt** [comment]
@nickvergessen ok, I guess it’s fine. Will check the UDP stuff.

**Pull request merged**